data/card_catalogue/anger.json

```diff
@@ -15,41 +18,50 @@
     "tier": "green",
     "details": {
       "rules": {
-	      "active": "A visible target is forced to stop what they're doing and discuss their feelings with you",
-	      "passive": "Whenever you make an attack, you can choose whether or not it will deal damage"
+	      "active": "A visible character is forced to stop what they're doing and discuss their feelings with you.",
```

Comment:
restricting to characters as opposed to targets i.e. not machines, animals, etc.